scipy/stats/boost/tests/test_boost_stats.py

```diff
@@ -0,0 +1,77 @@
+'''Boost stats tests.'''
+
+import numpy as np
+from scipy.stats import (
+    bernoulli as scipy_bernoulli,
+    beta as scipy_beta,
+    nbinom as scipy_nbinom,
+    binom as scipy_binom,
+    ncx2 as scipy_ncx2,
+)
+from scipy.stats.boost import (
+    bernoulli as boost_bernoulli,
+    beta as boost_beta,
+    nbinom as boost_nbinom,
+    binom as boost_binom,
+    ncx2 as boost_ncx2,
+)
+
+
+def test_issue_11026():
+    p = 0.5
+    assert boost_bernoulli(p).median() == 0
+    assert boost_bernoulli(p).interval(1) == scipy_bernoulli(p).interval(1)
+    assert boost_bernoulli(p).interval(0.999) == scipy_bernoulli(p).interval(0.999)
+    assert boost_bernoulli(p).interval(1-1e-14) == scipy_bernoulli(p).interval(1-1e-14)
```

Comment:
```suggestion
```
This doesn't really fix 11026, so I'm not sure this test is necessary. (I'm not actually sure SciPy is wrong.) Actually, unless we find another issue that Boost's `bernoulli` fixes, we might want to remove it from this PR. Don't pull the trigger yet, though; I'll look for other issues.

Reply:
`boost.bernoulli` distribution and tests have been removed